tests/libvmi/factory.go

```diff
@@ -29,42 +29,44 @@ import (
 const (
 	DefaultTestGracePeriod int64 = 0
 	DefaultVmiName               = "testvmi"
+	userDataSetPassword          = `#!/bin/bash
```

Comment:
Please separate from the above ones using a newline, these belong to different things.